src/client/components/ResultMessageModalWindow/Result.component.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import './ResultMessageModalWindow.styles.css';
+import PropTypes from 'prop-types';
+
+export default function Result({ correctAnswer }) {
```

Comment:
Try to avoid code duplication. DRY.

Reply:
sorry Neils, Dont understand what oyou mean her with (code duplication)... where I duplicated the code?

Reply:
There is duplication in the conditional. Only the h1 in the middle is different

Reply:
code duplication was resolved.. :+1: